BlockSettleUILib/Trading/RFQDialog.cpp

```diff
@@ -234,9 +234,8 @@ std::shared_ptr<bs::SettlementContainer> RFQDialog::newCCcontainer()
       connect(ccSettlContainer_.get(), &ReqCCSettlementContainer::cancelTrade
          , requestWidget_, &RFQRequestWidget::cancelCCTrade);
 
-      auto orderUpdatedCb = [qId = quote_.quoteId, ccContainer = ccSettlContainer_]
-         (const bs::network::Order& order)
-      {
+      // Do not make circular dependency, capture bare pointer
```

Comment:
For me it doesn't look like circular dependency - more like an extra hanging reference.